mlflow/store/artifact/databricks_artifact_repo.py

```diff
@@ -81,7 +93,7 @@ def _compute_num_chunks(local_file: os.PathLike, chunk_size: int) -> int:
     return math.ceil(os.path.getsize(local_file) / chunk_size)
 
 
-def _complete_futures(futures_dict):
+def _complete_futures(futures_dict, file):
```

Comment:
Can we isolate `tqdm` from the future completion logic?

Reply:
emmm we need to update progress_bar within the iteration of futures_dict, and the progress needs to be updated based on index & file_size. We can patch the 'future.result()' function but we still need to check if we want to use a pbar inside this function 🤔

Reply:
For example, we can `_complete_futures` a generator.